### PR TITLE
Remove the hardcoding for adding Intro text and wavey lines

### PR DESCRIPTION
**Pull request by g1039**
@saeedmb8 @miltontony Please review

### CODE REVIEW

**Comment by miltontony** [comment]
:+1:

File: tuneme/templates/core/article_page.html [hunk 1]
```diff
@@ -35,11 +35,6 @@
         <hr>
         <p class="bold">{{ block.value }}</p>
       {% else %}
-      {% if block.block_type == 'page' %}
-        <hr>
-        <div class="readmore">{% trans "You may want to read:" %}</div>
-        {{ block }}
-      {% else %}
```
Comment (smarzban) [inline tuneme/templates/core/article_page.html:42]: @TrevorOctober you don't need to remove the if block, you can just remove the `<hr>` and the `you may want to read` so you should have 

```
{% if block.block_type == 'page' %}
    {{ block }}
{% else %}
```

**Comment by g1039** [comment]
@saeedmb8 Ready for review

**Comment by smarzban** [comment]
:+1: once travis is happy

**Pull request merged**